libafl/src/state/mod.rs

```diff
@@ -600,7 +598,6 @@ impl<I, C, R, SC> HasClientPerfMonitor for StdState<I, C, R, SC> {
     }
 }
 
-#[cfg(test)]
```

Comment:
Why use NopState for reals?

Reply:
See how it is used to test in libafl_verilator -- `#[cfg(test)]` is not propagated to dependency crates.